avocado_vt/vt_agent/src/avocado_vt/agent/app/cmd.py

```
@@ -0,0 +1,179 @@
+    Write the current process ID to the PID file.
+
+    :param pid_file: Path to the PID file
+    :type pid_file: str
+    :raises OSError: If PID file cannot be written
+    """
+    pid = str(os.getpid())
+    try:
+        with open(pid_file, "w") as f:
+            f.write(pid + "\n")
+    except OSError as e:
+        LOG.error("Failed to write PID file %s: %s", pid_file, e)
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)